frontend/package.json

```diff
@@ -25,6 +26,7 @@
     "@lingui/react": "^2.7.2",
     "@reach/component-component": "^0.1.1",
     "@reach/router": "^1.2.1",
+    "acorn": "^6.0.4",
```

Comment:
Are we using this? I can't find it elsewhere in the code.

Reply:
I can't remember why that was in there. I've pushed another commit that removes it.